### PR TITLE
Add image for antiscam bot

### PR DESCRIPTION
**Pull request by ddorgan**

### CODE REVIEW

**Comment by ddorgan** [comment]
@fgimenez ok with adding this as an option to the statefulset?

File: charts/matrix-server/templates/statefulset.yaml [hunk 1]
```diff
@@ -90,6 +90,11 @@ spec:
           - configMap:
               name: matrix-server-antiscam-init
           {{ end }}
+      {{ if .Values.antiscam }}
+      - name: matrix-antiscam
+        image: {{ .Values.antiscam_image.repo }}:{{ .Values.antiscam_image.tag }}
```
Comment (fgimenez) [inline charts/matrix-server/templates/statefulset.yaml]: We need to define default values for this in `values.yaml`

Comment (ddorgan) [reply charts/matrix-server/templates/statefulset.yaml]: I was going to put those in the community-matrix-server repo, or would you prefer it defined here?

Comment (fgimenez) [reply charts/matrix-server/templates/statefulset.yaml]: We need some default values defined here, if not helm lint will fail and the chart won't be published.

Comment (ddorgan) [reply charts/matrix-server/templates/statefulset.yaml]: Looking ok now?

Comment (fgimenez) [reply charts/matrix-server/templates/statefulset.yaml]: indeed thx!

**Pull request merged**